docs/src/gcode/tool-compensation.adoc

```diff
@@ -248,13 +248,13 @@ The io program provides hal output pins to facilitate toolchanger management:
 
 LinuxCNC supports three types of tool changers: 'manual', 'random location'
 and 'non-random or fixed location'. Information about configuring a LinuxCNC tool changer
-is in the <<sec:emcio-section,EMCIO Section>> of the INI chapter.
+is in the <<sub:ini:sec:emcio,EMCIO Section>> of the INI chapter.
 
 .Manual Tool Changer
-Manual tool changer (you change the tool by hand) is treated like a fixed location tool changer.
-Using the manual tool changer only makes sense if you have toolholders that remain with the tool (Cat, NMTB, Kwik Switch, etc.) so that the tool position is preserved in the spindle.
-Machines with R8 or router collet type toolholders do not retain the tool location and the manual tool changer should not be used.
-Manual toolchanges can be aided by a hal configuration that employs the userspace program *hal_manualtoolchange* and is typically specified in an ini file
+Manual tool changer (you change the tool by hand) is treated like a
+fixed location tool changer.  Manual toolchanges can be aided by
+a hal configuration that employs the userspace program
+*hal_manualtoolchange* and is typically specified in an ini file
 with ini statements:
```

Comment:
Why did you remove/shorten this?

Reply:
I have absolutly no remembering and can not imagine myself do it !???

FIXED thanks